src/modules/LanItems.rb

```diff
@@ -2535,13 +2535,18 @@ def clear_set_hostname
     # @param [String] device type
     # @return [String] available device name
     def new_type_device(type)
+      new_type_devices(type, 1).first
+    end
+
+    def new_type_devices(type, count)
       raise ArgumentError, "Valid device type expected" if type.nil? || type.empty?
+      return [] if count < 1
```

Comment:
please document, as this behavior is not expected for me.

Reply:
ok